### ISSUE TITLE
Error on launch (Windows)

### ISSUE THREAD

**Issue by zddguu** [zenprotocol/zenwallet#64]
I downloaded and installed the 0.9.23 wallet on Windows 10
When I run the program, the window flashes up for a fraction of a second and then closes.
I have used an older version of the Wallet that worked (from December 2017, I think).
Running the wallet exe through the command prompt gives me the following error text:
```
***********************
[ZEN NODE]: Zen node args []
***********************


WARNING: You are using single-threaded workstation garbage collection, which means that parallel programs cannot scale.  Please configure your program to use server garbage collection.
[ZEN NODE]: Received WARNING: You are using single-threaded workstation garbage collection, which means that parallel programs cannot scale.  Please configure your program to use server garbage collection.
 bytes of data.
[[ZEN NODE]: Received [ bytes of data.
18:19:20 [ZEN NODE]: Received 18:19:20  bytes of data.
INF] [ZEN NODE]: Received INF]  bytes of data.
Node running... press CTRL+C to exit[ZEN NODE]: Received Node running... press CTRL+C to exit bytes of data.

[ZEN NODE]: Received
 bytes of data.
[18:19:20 INF] No tip in db
[ZEN NODE]: Received [18:19:20 INF] No tip in db
 bytes of data.
[18:19:20 INF] Listening on [ZEN NODE]: Received [18:19:20 INF] Listening on  bytes of data.
*:9655
[ZEN NODE]: Received *:9655
 bytes of data.

Unhandled Exception: System.Exception: 1455
   at Microsoft.FSharp.Core.PrintfModule.PrintFormatToStringThenFail@1645.Invoke(String message)
   at DataAccess.DatabaseContextModule.create(String pathToFolder) in C:\projects\zenprotocol\src\DataAccess\DatabaseContext.fs:line 51
   at Network.AddressBook.create(String dataPath) in C:\projects\zenprotocol\src\Network\AddressBook.fs:line 20
   at Network.Main.main@320-1.Invoke(T poller, IObservable`1 sbObservable, IObservable`1 ebObservable) in C:\projects\zenprotocol\src\Network\Main.fs:line 323
   at Infrastructure.Actor.create@13-3.Invoke(T shim) in C:\projects\zenprotocol\src\Infrastructure\Actor.fs:line 20
   at NetMQ.NetMQActor.RunShim()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
[ZEN NODE]: Closed
******* [will-quit] *******
Gracefully shutting down the zen-node
```

**Comment by goldylucks** [comment]
@somdoron ideas?

**Comment by zddguu** [comment]
This appears to be fixed in 0.9.26

**Comment by goldylucks** [comment]
@zddguu thanks for updating, closing
